server/src/main/java/com/linecorp/centraldogma/server/command/UpdateServerStatusCommand.java

```diff
@@ -21,32 +21,39 @@
 import javax.annotation.Nullable;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.MoreObjects.ToStringHelper;
 
 import com.linecorp.centraldogma.common.Author;
+import com.linecorp.centraldogma.server.management.ServerStatus;
 
 /**
  * A {@link Command} which is used to update the status of all servers in the cluster.
  */
+@JsonInclude(Include.NON_NULL)
 public final class UpdateServerStatusCommand extends AdministrativeCommand<Void> {
 
-    private final boolean writable;
+    private final ServerStatus serverStatus;
 
+    /**
+     * Creates a new instance with the specified properties.
+     */
     @JsonCreator
-    UpdateServerStatusCommand(@JsonProperty("timestamp") @Nullable Long timestamp,
-                              @JsonProperty("author") @Nullable Author author,
-                              @JsonProperty("writable") boolean writable) {
+    public UpdateServerStatusCommand(@JsonProperty("timestamp") @Nullable Long timestamp,
+                                     @JsonProperty("author") @Nullable Author author,
+                                     @JsonProperty("serverStatus") ServerStatus serverStatus) {
```

Comment:
Is there any backward compatibility / gradual rollout concern?

Reply:
The command isn't used yet internally. Are you concerned that some users might have used the command already in a plugin?

Reply:
Nope. I guess we're the only user at the moment :sweat_smile: